packages/react-storage/src/components/StorageBrowser/controls/hooks/useTitle.ts

```diff
@@ -0,0 +1,21 @@
+import { TitleProps } from '../../composables/Title';
+import { useControlsContext } from '../../controls/context';
+import { CLASS_BASE } from '../../views/constants';
+
+const BLOCK_NAME = `${CLASS_BASE}__title`;
+
+export type UseTitle = () => {
+  props?: TitleProps;
+};
+
+export const useTitle: UseTitle = () => {
+  const { data } = useControlsContext();
+  const { title } = data;
+
+  return {
+    props: {
+      children: data?.title ? title : '',
```

Comment:
Nit: I think this should be safe?
```suggestion
      children: data?.title,
```

Reply:
I see, also removed `{ title } ` above as it's not needed if we stick to it